tst/EnergyPlus/unit/InternalHeatGains.unit.cc

```diff
@@ -130,7 +130,7 @@ TEST_F(EnergyPlusFixture, InternalHeatGains_OtherEquipment_CheckFuelType)
         if (equip.Name == "OTHEREQ1") {
             ASSERT_EQ(equip.OtherEquipFuelType, 0);
         } else if (equip.Name == "OTHEREQ2") {
-            ASSERT_EQ(equip.OtherEquipFuelType, ExteriorEnergyUse::PropaneUse);
+            ASSERT_EQ(equip.OtherEquipFuelType, state.exteriorEnergyUse.PropaneUse);
```

Comment:
Ahh, and here is one reason why we won't be able to put that enum into the cc file, because the unit test needs to test against it.  That's fine, the enum can go in the header, but in general, it's nice to move things out of the header that don't need to be there.